obspy/io/nordic/ellipse.py

```diff
@@ -0,0 +1,329 @@
+"""
```

Comment:
This file should have the standard obspy headers, see the content of other source files.

Reply:
Done (I think)

Reply:
You'll need the licence and copyright up the top I think:
```python
:copyright:
    The ObsPy Development Team (devs@obspy.org)
:license:
    GNU Lesser General Public License, Version 3
    (https://www.gnu.org/copyleft/lesser.html)
```

Reply:
Done